src/ticks.h

```diff
@@ -103,6 +103,7 @@ with simple integral values. The following describes these calculations:
 #endif
 
 #include "ticks_os.h"
+#include "os_abstract.h"
```

Comment:
Same here, I see that you use `os_thread_t` in file `client.cpp`. Why not include header `os_abstract.h` in `client.h`?

Reply:
This is one of the changes required to properly build in alpine OS, the program was not able to find this file